src/scenes/Question/scenes/Answer/components/Source/index.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import Input from 'react-toolbox/lib/input/Input'
+import Button from 'react-toolbox/lib/button/Button'
+import FontIcon from 'react-toolbox/lib/font_icon/FontIcon'
+
+const Source = props => {
+  return (
+    <div
+      style={{ display: 'flex', flexDirection: 'row', alignItems: 'center' }}
+    >
+      <div>
+        <FontIcon value="library_books" style={{ marginRight: '1rem' }} />
+      </div>
+      <div style={{ marginRight: '1rem', width: '40%' }}>
+        <Input
+          type="text"
+          label="Label"
+          value={props.source.label}
+          onChange={props.changeLabel}
+        />
+      </div>
+      <div style={{ flex: '1' }}>
+        <Input
+          type="text"
+          label="URL"
+          value={props.source.url}
+          onChange={props.changeUrl}
+        />
+      </div>
+      <div style={{ marginLeft: '1rem' }}>
+        <Button
+          icon={
+            <i
+              className="material-icons"
+              style={{ marginLeft: '6px', paddingTop: '6px' }}
+            >
+              close
+            </i>
+          }
+          style={{ minWidth: 'initial', alignItems: 'center' }}
+          accent
+          onClick={() => props.delete()}
```

Comment:
Why don't you pass `delete` directly ? `onClick={props.delete}`

Reply:
Simple oversight after a refact, will fix!